app/assets/javascripts/channels/main_page.js

```diff
@@ -0,0 +1,36 @@
+$(function () {
+
+  'use strict';
+
+  var organization = $('#current_organization').text().trim();
+
+  if (location.pathname === "/" && location.search === '') {
+    var current_user_id = $('.current_user_id').data('id');
+    App.cable.subscriptions.create({
+      channel: "MainPageChannel",
+      organization: organization
+    }, {
+      received: function(data) {
+        if (!(data['user_id'] === current_user_id)) {
```

Comment:
@shir I don't work with vanilla javascript very long time, I just forgot how to do this in js :)